libs/sdk-backend-tiger/src/convertors/toBackend/afm/FilterConverter.ts

```diff
@@ -156,6 +160,18 @@ export function convertMeasureValueFilter(filter: IMeasureValueFilter): MeasureV
     return null;
 }
 
+export function convertRankingFilter(filter: IRankingFilter): RankingFilter {
+    const { measure, attributes, operator, value } = filter.rankingFilter;
+    return {
+        rankingFilter: {
+            measures: [toMeasureValueFilterMeasureQualifier(measure)],
+            dimensionality: attributes?.map(toRankingFilterDimensionalityIdentifier),
```

Comment:
just checking: this means dimensionality may be undefined / null when in the request body. is that OK from tiger perspective? 

or perhaps there should not be any `dimensionality` property in case there are no attributes?

Reply:
Good catch! Indeed the backend validator doesn't allow explicit null here. The question is whether it shouldn't but I suggest to leave it as is and just fix the SDK code.